lib/text/lrc_text_parser.js

```diff
@@ -0,0 +1,121 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.text.LrcTextParser');
+
+goog.require('goog.asserts');
+goog.require('shaka.log');
+goog.require('shaka.text.Cue');
+goog.require('shaka.text.TextEngine');
+goog.require('shaka.util.StringUtils');
+
+
+/**
+ * @implements {shaka.extern.TextParser}
```

Comment:
Would you please add a short description here that includes a link to the format description on Wikipedia?
 https://en.wikipedia.org/wiki/LRC_(file_format)

Reply:
Done!